paddle/math/tests/test_matrixCompare.cpp

```diff
@@ -1203,4 +1203,120 @@ TEST(Matrix, warpCTC) {
   }
 }
 
+int outputSizeCol2Vol(
```

Comment:
same as above comments.  `./paddle/math/MathUtils.h` has contains the calculation.